### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by curquiza**
This is a draft until all the parts are completed.
@eskombro will complete the `WIP` parts by doing a PR pointing to this PR branch.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+2. **You've read the MeiliSearch [documentation](https://docs.meilisearch.com) and the [README](/README.md).**
+3. **You know about the [MeiliSearch community](https://docs.meilisearch.com/resources/contact.html). Please use this for help.**
+
+## How to Contribute
+
+1. Make sure that the contribution you want to make is explained or detailed in a GitHub issue! Find an [existing issue](https://github.com/meilisearch/meilisearch-digitalocean/issues/) or [open a new one](https://github.com/meilisearch/meilisearch-digitalocean/issues/new).
+2. Once done, [fork the meilisearch-digitalocean repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) in your own GitHub account. Ask a maintainer if you want your issue to be checked before making a PR.
+3. [Create a new Git branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository).
+4. Review the [Development Workflow](#workflow) section that describes the steps to maintain the repository.
```
Comment (eskombro) [inline CONTRIBUTING.md:25]: I can't find the "workflow" section. So this link isn't working. Should the line be removed? Or the link updated?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+We don't follow any other convention, but if you want to use one, we recommend [this one](https://chris.beams.io/posts/git-commit/).
+
+### GitHub Pull Requests
+
+Some notes on GitHub PRs:
+
+- [Convert your PR as a draft](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/changing-the-stage-of-a-pull-request) if your changes are a work in progress: no one will review it until you pass your PR as ready for review.<br>
+  The draft PR can be very useful if you want to show that you are working on something and make your work visible.
+- The branch related to the PR must be **up-to-date with `master`** before merging. Fortunately, this project [integrates a bot](https://github.com/meilisearch/integration-guides/blob/master/guides/bors.md) to automatically enforce this requirement without the PR author having to do it manually.
```
Comment (eskombro) [inline CONTRIBUTING.md:53]: Are we adding BORS to this repo?

**Review by eskombro — COMMENTED** [review]
I'll do a PR on the `Release Process` to complete your amazing work, just had 2 questions on the main repo guides

File: CONTRIBUTING.md (hunk 1 above)
Comment (curquiza) [reply CONTRIBUTING.md:25]: Oh yes! This should be updated! Nice catch!

File: CONTRIBUTING.md (hunk 2 above)
Comment (curquiza) [reply CONTRIBUTING.md:53]: We probably won't, so for the moment I will remove it, thanks!!

**Comment by curquiza** [comment]
I've done the change @eskombro! Tell me if it's ok for you!

**Review by eskombro — COMMENTED** [review]
Evetything looks good to me! I'll start that PR now, so we can merge this!

**Comment by curquiza** [comment]
I wish I can approve our work but I cannot. LGTM anyway 😊

**Review by eskombro — APPROVED** [review]

**Pull request merged**